src/tribler-core/tribler_core/components/restapi/restapi_component.py

```diff
@@ -41,6 +42,9 @@
 from tribler_core.utilities.unicode import hexlify
 
 
+warnings.filterwarnings("ignore", "Multiple schemas resolved to the name SpeedTestRequest", module=".*marshmallow.*")
```

Comment:
Suggestion: I think we can fix this one by updating the IPv8 pointer (see [this](https://github.com/Tribler/py-ipv8/commit/56c4b2dc37f7498754a69f577325f01ab0ad9d7a) commit).